source/administration/config-settings.rst

```diff
@@ -3756,6 +3758,30 @@ The number of milliseconds to leave an idle connection open between servers in t
 | This feature's ``config.json`` setting is ``"IdleConnTimeoutMilliseconds": 90000`` with numerical input.                                                             |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Network Interface
+^^^^^^^^^^^^^^^^^
+An IP address used identify the device that does automatic IP detection in high availability clusters. 
+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
Tables are not rendering. You can choose "No wrap" in the top right corner to easily format tables.

Reply:
Thanks @amyblais Should be good to go now.